tools/src/vendoring/legacy.ts

```diff
@@ -18,6 +18,9 @@ interface VendoredModuleUpdateStep {
   targetAndroidPackage?: string;
   recursive?: boolean;
   updatePbxproj?: boolean;
+
+  // a hook for a file did finished for vendoring
+  afterVendoredHookAsyncAndroid?: (file: string) => Promise<void>;
```

Comment:
The name is soo confusing that I'm completely not sure what to expect from this hook 😅 
I think we can express more general idea by this hook.
```suggestion
  /**
   * Hook that is fired by the end of vendoring an Android file.
   * You should use it to perform some specialistic operations that are not covered by the main flow. 
   */
  onDidVendorAndroidFile?: (file: string) => Promise<void>;
```
Alternatively we can go with `onAfterVendoredAndroidFile`, but I remember that we've decided to follow iOS's style with `will<verb>` and `did<verb>` 🤔

Reply:
that makes sense to me, will revise that. thanks for the comment